_posts/2020-06-20-snap-minis.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Snap Minis: How we can help you bring your app into SnapChat"
+date: 2020-06-20 11:00:00
+reviewed: 2020-06-20 11:00:00
+categories: []
+author: ABizzinotto
+---
+
+Last week, [Snap](https://www.snap.com) announced a new product during the [Snap Partner Summit 2020](https://www.youtube.com/watch?v=BfvubHha69k), [Snap Minis](https://minis.snapchat.com/). Minis are bite-sized third-party applications that live inside [Snapchat](https://www.snapchat.com/) and can be accessed through Snapchat's Chat interface. They are built on the same principle and technology as the Snap Games and are easy to develop, allowing you to have an app that takes advantage of the existing [Snapchat](https://www.snapchat.com/) application and is available to Snapchat users worldwide.
```

Comment:
The chat interface is not the only way to open a mini, you can use the search on the top to access minis too.

Reply:
@arielj the issue here is we are writing a general article on Minis based on the information Snap has released as we don't want issues. That is the language Snap used in their communications. I'm not disclosing anything beyond that at this point. Later on when we write a case study, we can include more detailed information.